doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "dB, dBm, dBi, and dBd"
+section: Additional Information
+description: ""
+weight:
+---
+
+In this chapter we learn about some units in brief that are used to measure the performance of transceivers (gateways and end devices) and antennas.
+
+## dB (decibel)
+
+The decibel can be used to express the ratio of two physical quantities such as power, sound intensity, sound pressure, voltage, and current on a logarithmic scale. In LoRaWAN we use decibel to express the ratio between two power levels usually given in Watt (W) or milliwatt (mW).
+
+The power ratio can be expressed in decibels using the formula, 
+
+**dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>)**
+
+where P<sub>out</sub> is the output power and P<sub>in</sub> is the input power.
+
+{{< note "When we are dealing with the power levels we use 10log units." />}}
+
+For example, if an amplifier turns a 1 W signal into a 1000 W signal, its power ratio can be expressed as,
+
+dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>) = 10 log<sub>10</sub> (1000/1) = 30 dB
```

Comment:
Looking at this equation now, it looks like you had an idea to express the power ratio as "dB", but you can't do that, it's too confusing. It doesn't make sense to have dB = 30 dB. If you consider what I wrote above, you could say N_dB=....=30 dB

Reply:
@nejraselimovic  equation updated